homeassistant/components/ticktick/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "ticktick",
+  "name": "TickTick",
+  "codeowners": ["@arne182"],
+  "config_flow": true,
+  "iot_class": "cloud_polling",
+  "requirements": [],
```

Comment:
Please try to extract much of the 3900+ LOC of integration with the TickTick API which now lives in `__init__.py` to a separate library which should have it's source code in a public repo (for example github) and be released on PyPi.